dataactvalidator/config/sqlrules/b11_object_class_program_activity_1.sql

```diff
@@ -4,4 +4,5 @@ SELECT
 FROM object_class_program_activity AS op
 WHERE op.submission_id = {}
 AND op.object_class NOT IN (SELECT object_class_code FROM object_class)
-AND op.object_class <> '000'
+AND op.object_class <> '0'
+AND op.object_class <> '00'
```

Comment:
We still want to look for `af.object_class <> '000'` since it's the same warning as before